modules/integrate-circle-ci.adoc

```diff
@@ -0,0 +1,64 @@
+// Module included in the following assemblies:
+//
+// * integration/integrate-with-ci-systems.adoc
+:_module-type: PROCEDURE
+[id="integrate-circle-ci_{context}"]
+= Using CircleCI
+
+You can integrate {product-title} with CircleCI.
+
+.Prerequisetes
+* You must have a token with `read` and `write` permissions for the `Image` resource.
+* Useranme and password of your DockerHub account.
+
+.Procedure
+. Log in to CircleCI and open an existing project or create a new project.
+. Click *Project Settings*.
+. Click *Environment variables*.
+. Click *Add variable* and create the following three environment variables:
+** *Name*: *STACKROX_CENTRAL_HOST* - The DNS name or IP address of Central.
+** *Name*: *ROX_API_TOKEN* - The API token to access {product-title}.
+** *Name*: *DOCKERHUB_PASSWORD* - The password for your DockerHub account.
+** *Name*: *DOCKERHUB_USER* - The username for your DockerHub account.
+. Create a directory called `.circleci` in the root directory of your local code repository for your selected project, if you do not already have a CircleCI configuration file.
+. Create a `config.yml` configuration file with the following lines in the `.circleci` directory.
```

Comment:
Can you please change the period to a colon after directory? That's generally how we introduce an example code block.